src/cache.cpp

```diff
@@ -1100,7 +1100,7 @@ void Cache::fetch_descriptions(RssFeed* feed)
 	run_sql(query, fill_content_callback, feed);
 }
 
-std::string Cache::fetch_description(RssItem* item)
+std::string Cache::fetch_description(const RssItem* item)
```

Comment:
It might be useful to mark `RssItem` as a const reference because there is no checking for NULL.

Reply:
Definitely! Done in 3a9dc0a.